igvc_ws/src/igvc_nav/src/mt_dstar_node.py

```diff
@@ -114,17 +211,38 @@ def c_space_callback(c_space):
         # path_msg.data = data
         # path_pub.publish(path_msg)
 
+        if SHOW_PLOTS:
```

Comment:
This applies to all the SHOW_PLOTS blocks: It would be good to label what plots are in these blocks so I don't have to just know the answer.

I like the SHOW_PLOTS flag though, that's a nice touch